services/121-service/src/payments/fsp-integration/safaricom/safaricom.service.ts

```diff
@@ -34,6 +34,7 @@ export class SafaricomService
    * Do not use! This function was previously used to send payments.
    * It has been deprecated and should not be called anymore.
    */
+  // ##TODO: remove this method. What is the point of keeping it?
```

Comment:
Remove comment. We need to keep it for now because of "implements FinancialServiceProviderIntegrationInterface". When we refactor the PaymentsService, we can remove this.